### PR TITLE
[BUGFIX] Carousel Part component

### PR DESCRIPTION
**Pull request by dtiwarATS**
-- updated tag name for the Carousel component. It should be 'janus-image-carousel' not 'janus-carousel'

### CODE REVIEW

File: assets/src/components/parts/janus-Carousel/manifest.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {
-  "id": "janus_carousel",
+  "id": "janus_mage_carousel",
```
Comment (bsparks) [inline assets/src/components/parts/janus-Carousel/manifest.json:2]: missing i

Comment (dtiwarATS) [reply assets/src/components/parts/janus-Carousel/manifest.json]: Ooops... Done

**Review by bsparks — APPROVED** [review]

File: priv/repo/seeds.exs [hunk 2]
```diff
@@ -106,7 +106,7 @@ Oli.Registrar.register_local_part_components(
     "janus_audio",
     "janus_video",
     "janus_slider",
-    "janus_carousel",
+    "janus_image_carousel",
```
Comment (darrensiegel) [inline priv/repo/seeds.exs:109]: Renaming this part component is not sufficient to allow existing Torus instances to pick up this change.  This renaming works for seeding new databases, but existing ones will end up having the old "janus_carousel' and a new 'janus_image_carousel' both exist as part components due to how `Oli.PartComponents.register_part_component' works.  

A database migration is necessary to do this "renaming" correctly.  The migration can simply delete the 'janus_carousel' record, allowing the seed.ex change to drive the creation of a new one with the updated id.

**Review by darrensiegel — CHANGES_REQUESTED** [review]

**Comment by dtiwarATS** [comment]
Closing this PR for now. Once the migration script is ready and the rendering issues fixed then I will create another PR.

**Pull request closed without merging**